zuul.d/project-templates.yaml

```diff
@@ -775,6 +775,33 @@
       queue: integrated
       jobs: *ansible-collections-ansible-netcommon-units-jobs
 
+- project-template:
+    name: ansible-collections-ansible-network-units
+    check:
+      jobs: &ansible-collections-ansible-network-units-jobs
+        - ansible-changelog-fragment
+        - ansible-test-units-netcommon-python27
+        - ansible-test-units-netcommon-python35
+        - ansible-test-units-netcommon-python36
+        - ansible-test-units-netcommon-python37
+        - ansible-test-units-netcommon-python38
+
+        - build-ansible-collection:
```

Comment:
when you respin, can you sort this list.

We also need vyos.vyos

Reply:
updated!